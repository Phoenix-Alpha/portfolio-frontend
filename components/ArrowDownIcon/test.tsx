import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ArrowDownIcon } from '.'

describe('<ArrowDownIcon />', () => {
  it('should renders correct <ArrowDownIcon />', () => {
    const wrapper = shallow(<ArrowDownIcon />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
