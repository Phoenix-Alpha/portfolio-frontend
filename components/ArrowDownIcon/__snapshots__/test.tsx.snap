// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowDownIcon /> should renders correct <ArrowDownIcon /> 1`] = `
<svg
  className="arrow-down-icon"
  fill="none"
  height={6}
  viewBox="0 0 12 6"
  width={12}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M1.5 0.75L6 5.25L10.5 0.75"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="1.5"
  />
</svg>
`;
